UM/PluginRegistry.py

```diff
@@ -31,6 +32,10 @@ def __init__(self):
         self._application = None
         self._active_plugins = []
 
+        preferences = Preferences.getInstance()
+        preferences.addPreference("general/disabled_plugins", "")
+        self._disabled_plugins = None
```

Comment:
It gets reinitialised to a list here: https://github.com/fieldOfView/Uranium/blob/3886d9ff2900c2e9d09f78ec9c3416fee0ebdf4f/UM/PluginRegistry.py#L88

At the time the pluginregistry is initialised, the preferences may not yet be available.